_newsletters/2021-01-01-january-2021.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: January 2021 Newsletter
+category: newsletters
+last_modified_at: 
+tags: newsletters
+newsletter-year: 2021
+sidenav: about_us
+---
+
+## Coming Soon!
+### Search.gov 2020 Year In Review
+
+Last year we published our Year In Review infographic - it was so well received that this year we decided to do a deeper dive. It's almost done, so please watch your inbox in the near future. We're excited to share it with you!
+
+## SEO Pro Tip for Drupal Users
+### Simple Sitemap vs. XML Sitemap Module
+
+As more teams are working on site redesigns and upgrades, we wanted to alert Drupal users to some important differences between the available XML sitemap modules. In order to support search engine indexing processes, all the files that you want to appear in search results should be listed in the XML sitemap. This includes PDFs, docs, etc. If you’re new to XML sitemaps, please [read our explainer page](https://search.gov/manual/sitemaps.html).
+
+In Drupal 7, file attachments needed to be added to the XML sitemap manually, one by one. This is still the case in later versions of Drupal if you’re using the Simple XML Sitemap module. For many teams, this is not a sustainable way to ensure PDFs appear in search results.
+
+We recommend that all Drupal sites use the [XML Sitemap module](https://www.drupal.org/project/xmlsitemap), if possible: it has a similar name, but different features. This module allows you to include file attachments globally by type, with a simple check box within the module settings, via the Sitemap Entities tab.
+
+![Screen Shot 2021-01-29 at 10.46.21 AM](https://cdn2.hubspot.net/hub/1962994/hubfs/Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png?upscale=true&width=1000&upscale=true&name=Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png)
```

Comment:
do we have a native version of this?

Reply:
don't want the image reference to break. so we should either create a version of the file on our site, or remove from the newsletter.

Reply:
added the image file